x-pack/plugins/case/server/connectors/case/index.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { curry } from 'lodash';
+
+import { KibanaRequest } from 'kibana/server';
+import { ActionTypeExecutorResult } from '../../../../actions/common';
+import { CasePatchRequest, CasePostRequest } from '../../../common/api';
+import { createCaseClient } from '../../client';
+import { CaseExecutorParamsSchema, CaseConfigurationSchema } from './schema';
+import {
+  CaseExecutorParams,
+  CaseExecutorResponse,
+  ExecutorSubActionCreateParams,
+  ExecutorSubActionUpdateParams,
+  ExecutorSubActionAddCommentParams,
+  CaseActionType,
+  CaseActionTypeExecutorOptions,
+} from './types';
+
+import { GetActionTypeParams } from '..';
+
+const supportedSubActions: string[] = ['create', 'update', 'addComment'];
+
+// action type definition
+export function getActionType({
+  logger,
+  caseService,
+  caseConfigureService,
+  userActionService,
+}: GetActionTypeParams): CaseActionType {
+  return {
+    id: '.case',
+    minimumLicenseRequired: 'gold',
+    name: 'Case',
```

Comment:
`i18n` for name please

Reply:
I think that the backend does not support i18n, right?

Reply:
i looked in the code at other instances of `getActionType(` and they all have i18n for this field.